server/core/config/raw/global_cfg.go

```diff
@@ -75,7 +74,6 @@ func (g GlobalCfg) Validate() error {
 		validation.Field(&g.PullRequestWorkflows),
 		validation.Field(&g.DeploymentWorkflows),
 		validation.Field(&g.Metrics),
-		validation.Field(&g.Jobs),
```

Comment:
Maybe don't remove this and instead just pipe this into the new persistence object? This is so that you don't have to worry about breaking deploys when this goes out automatically without us changing our repo config to match.